docs/install.md

```diff
@@ -1302,6 +1314,45 @@ deployment is in the `Failed` state, then one can look into the event(s) using
 > [`DeploymentSpec`](#deployment-crd-api).
 
 
+## PMEM-CSI operator API conversion webhook
+
+The operator relese v0.9.0 deprecates the `v1alpha1` [deployment API](#pmem-csi-deployment-crd) infavour of
+`v1beta1`. To support auto-migrating of deprecated(v1alpha1) API versions, The
```

Comment:
Please explain what "auto-migrating" does. Be specific (i.e. provide examples) to illustrate it. This is also the right place to describe what happens without the webhook. We have to assume that the reader is not familiar with CRD concepts.

Reply:
Added more explanation about CRD conversion and added suitable examples to the docuentation.